R/app_ui.R

```diff
@@ -11,7 +11,8 @@ app_ui <- function(request) {
     # Your application UI logic
     fluidPage(
       h1("Old Faithful Geyser - Hist"),
```

Comment:
```suggestion
      h1("Old Faithful Geyser - Hist"),
```
      h1("Old Faithful Geyser - Histograms"),